dags/analyzer_helper/discord/discord_transform_raw_data.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from dags.analyzer_helper.discord.transform_raw_data_base import TransformRawDataBase
+
+
+class DiscordTransformRawData(TransformRawDataBase):
+    def transform(self, raw_data: list, platform_id: str, period: datetime) -> list[dict]:
+        transformed_data = []
+        for data in raw_data:
+            interactions = []
+            if data.get("replied"):
+                interactions.append({
+                    "name": "reply",
+                    "users_engaged_id": [data.get("replied")],
+                    "type": "receiver"
+                })
+
+            if data.get("user_mentions"):
+                interactions.append({
+                    "name": "mention",
+                    "users_engaged_id": data.get("user_mentions"),
+                    "type": "emitter"
+                })
+
+            if data.get("reactions"):
+                for reaction in data.get("reactions"):
+                    parts = reaction.split(", ")
```

Comment:
The users are splited with just a comma (`,`) no space is required. So line 27 would be like below

```
parts = reaction.split(",")
```